### PR TITLE
fix CPU interpolate bug and add related test case

### PR DESCRIPTION
**Pull request by meiyang-intel**
### Details:
 - *Fix CPU interpolate bug and add related test case*

### Tickets:
 - *CVS-97985*

### CODE REVIEW

**Comment by yuxu42** [comment]
@chenhu-wang  Could you please take a review? Thanks!

**Comment by chenhu-wang** [comment]
could you please update bin_cov as well here: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/bin_conv.cpp#L589-L590

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp [hunk 1]
```diff
@@ -331,6 +337,8 @@ const std::vector<fusingSpecificParams> interpolateFusingParamsSet{
         emptyFusingSpec,
 #if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
         fusingSwish,
+        fusingAddPerTensor,
+        fusingMultiplyPerTensor,
```
Comment (chenhu-wang) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp:341]: we need per-channel post-ops to cover this fix.

Comment (chenhu-wang) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp:342]: let's remove this two lines as per-tensor post-ops is already covered, which doubled smoke test time. Interpolate smoke test and nightly test is heavy.

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp [hunk 2]
```diff
@@ -462,13 +471,19 @@ const std::vector<ShapeParams> shapeParams4D_fixed_C = {
 };
 
 #if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
+const std::vector<fusingSpecificParams> interpolateFusingParamsSet_fixed_C{
+        fusingFakeQuantizePerChannelRelu,
+        fusingAddPerChannel,
+        fusingMultiplyPerChannel,
```
Comment (chenhu-wang) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp:477]: leave one per-channel depthwise post-ops is enough;-)

**Comment by meiyang-intel** [comment]
> could you please update bin_cov as well here: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/bin_conv.cpp#L589-L590

Done! But there is no single layer test for bin_conv. Have created a ticket to track it. https://jira.devtools.intel.com/browse/CVS-120644

**Review by chenhu-wang — APPROVED** [review]

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**